src/reducers/contacts.reducer.spec.js

```diff
@@ -0,0 +1,147 @@
+import {
+  increaseLikes,
+  decreaseLikes,
+  deleteContact,
+  deleteSelectedContacts,
+  clearContacts,
+  changeIsChecked,
+  fetchFavourites,
+  fetchContacts,
+  addContact
+} from '../actions';
+import contactsReducer from './contacts.reducer';
+import randomContactsList from '../data/data';
+import { getRandomNumber, getFavouritesList } from '../helpers/helper';
+import { DEFAULT_NUMBER_OF_CONTACTS as N } from '../constants/constants';
+
+describe('contacts reducer with id param', () => {
+
+  const state = {
+    contacts: randomContactsList,
+    favourites: []
+  };
+  let randomContactIndex;
+  let randomContact;
+
+  beforeEach(() => {
+    randomContactIndex = getRandomNumber(0, N);
+    randomContact = state.contacts[randomContactIndex];
+  });
+
+  test('increased likes', () => {
+    const newState = contactsReducer(state, increaseLikes(randomContact.id));
+    expect(newState.contacts[randomContactIndex].likes).toEqual(randomContact.likes + 1);
+    expect(newState).toEqual({
```

Comment:
you don't need this expect since you test increase likes. the first expect should be enough